s3_file_field/_multipart_boto3.py

```diff
@@ -16,13 +16,19 @@ def __init__(self, storage: 'S3Boto3Storage'):
         self._client: s3.Client = resource.meta.client
         self._bucket_name: str = storage.bucket_name
 
-    def _create_upload_id(self, object_key: str) -> str:
+    def _create_upload_id(
+        self,
+        object_key: str,
+        content_type: Union[str, None] = None,
+    ) -> str:
+        boto3_kwargs = {}
```

Comment:
Since we know the file name at this point, can we set `ContentDisposition` too?

Reply:
We could, but AWS already provides a sensible default value when it is not set. I don't think at this point we would be able to make a better guess than that.